src/app/hero-list/hero-list.component.ts

```diff
@@ -27,29 +28,46 @@ export class HeroListComponent implements OnInit {
     })
     );
   }
-  getNextHeroes(pageNumber: number): void {
-    this.getHeroes(pageNumber + 1).subscribe(() => {
-      this.pageNumber += 1;
-    })
-  }
-  getPrevHeroes(pageNumber: number): void {
-    this.getHeroes(pageNumber - 1).subscribe(() => {
-      this.pageNumber -= 1;
-    })
-  }
 
   handleNextPage(): void {
```

Comment:
I have two functions (handleNextPage and handlePreviousPage) which have almost same code. The only difference is that one has '+1' and the other has '-1'. Maybe it's better idea to make one function that takes argument like
`pageDifference: number` and just call it like `handlePageChange(1)` and `handlePageChange(-1)`?